src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] defaultFigures = figureSupplier.getDefaultFigure();
```

Comment:
The method name `getDefaultFigure` suggests that it returns a single figure, but here it seems to be used as if it returns an array of figures. The method should be correctly named to reflect what it returns, for example, `getDefaultFigures` if it returns an array.